k8s.gcr.io/k8s-staging-csi/manifest.yaml

```diff
@@ -0,0 +1,9 @@
+# google group for gcr.io/k8s-staging-csi is k8s-infra-gcr-staging-csi@googlegroups.com
+registries:
+  src: gcr.io/k8s-staging-csi
+  dest: gcr.io/k8s-staging
+service-account: service-account@kubernetes-public.iam.gserviceaccount.com
+#images:
```

Comment:
Can you uncomment this line? The `images` key is part of a Go struct so the marshaling logic will break if it is missing IIRC.

Reply:
Err, might as well make that `images: []`.

Reply:
ack